libsolidity/inlineasm/AsmStack.cpp

```diff
@@ -44,6 +49,11 @@ bool InlineAssemblyStack::parse(shared_ptr<Scanner> const& _scanner)
 	return true;
 }
 
+string InlineAssemblyStack::print()
```

Comment:
Shouldn't this be called something like `toFormattedString()`?  It is not printing anything.